demaceo/binary
# Display To-Dos

#### What does  this PR do?
- Creates **ToDo** component.
- Corrects how data / props were being passed through .
- Import _PropTypes_ to various components.
- Displays each **ToDo** component in within the **ToDos** component.
- Color coordinates each **ToDo** by priority.
- Fixes issue with _localStorage_.
- Delete **Date** & **Clock** components.
- Comments out all functionality for **Category** component.
#### How should this be manually tested?
1. Click the '+' icon in the **NavBar** component
2. Type in the input, choose a priority level, & click the button.
3. Go to the **ToDos** component to see the **ToDo** displayed
#### Any background context you want to provide?
- Need to disable the button if the input field is empty.
#### What are the relevant tickets?
Closes #21
